shading/coral-trino-parser/build.gradle

```diff
@@ -22,8 +18,8 @@ shadowJar {
   dependsOn(relocateShadowJar)
 }
 
-assemble.dependsOn shadowJar
-
-artifacts {
-  archives shadowJar, javadocJar, sourcesJar
+// Rename shadow jar to original artifact name, so that 'coral-trino-parser-x.x.x.jar' itself is the shadow jar
```

Comment:
https://imperceptiblethoughts.com/shadow/configuration/#configuring-output-name

try out the instructions from the manual for renaming the shadow lib